config/routes.rb

```diff
@@ -1,10 +1,10 @@
 Sprangular::Engine.routes.draw do
   scope module: 'sprangular' do
     root to: 'home#index'
+    resources 'locale', only: :show, param: :locale
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.